github/handle_github.py

```diff
@@ -90,7 +90,13 @@ def check_issue(self, num):
                 print(error)
                 self.error_string.append(error)
                 return False
-            for label in issue.labels:
+            # compatibility between 1.20 and 0.9.6
+            if type(issue.labels) == type([]):
+                l = issue.labels
+            else:
+                l = issue.labels()
+
+            for label in l:
```

Comment:
If issue.labels is not list or not of any iterable type, it will not iterate. Here in newer version it is issue.labels().

Reply:
issue.label on newer github3.py is a function that return a iterable and a list on the old version, so I am not sure if I should change something or not ?

Reply:
Now I see your point. 
LGTM